xbmc/interfaces/builtins/LibraryBuiltins.cpp

```diff
@@ -48,10 +48,16 @@ static int CleanLibrary(const std::vector<std::string>& params)
   bool userInitiated = true;
```

Comment:
As you use `param[0]` very often should we assign `param[0]` to a local variable e.g. `std::string content = param[0]`

Reply:
how many strings named content do you want ;) in any case this violates my ODR rule. param[0] tells me much more than some random string named content. for instance, i immediately know it is a parameter to the built-in that is externally provided through the generic context. if it was put into a temporary i would have to read more code to understand its origin. so i disagree and won't do this unless you insist.

Reply:
Okay leave it